libpod/container_internal.go

```diff
@@ -1082,8 +1082,11 @@ func (c *Container) addLocalVolumes(ctx context.Context, g *generate.Generator,
 		if MountExists(g.Mounts(), k) {
 			continue
 		}
-		volumePath := filepath.Join(c.config.StaticDir, "volumes", k)
-
+		volume, err := c.runtime.NewVolume(ctx, []VolumeCreateOption{}...)
```

Comment:
We need to give the volume a name, at least - something based on the container's name. containername-volumepath?

Reply:
The volume gets named automatically if there is no name specified.  This is how it works in Docker also.  The problem with us naming it, is that there could be multiple built in volumes within an image.